src/main/java/edu/oregonstate/cs361/battleship/BattleshipModel.java

```diff
@@ -84,25 +83,51 @@ public BattleshipModel() {
 
         scanResult = false;
     }
-
+// Refactored getShip into a case statement instead of a mass of cluttered if statements. -GH
     public Ship getShip(String shipName) {
-        if (shipName.equalsIgnoreCase("aircraftCarrier")) {
-            return aircraftCarrier;
-        } if(shipName.equalsIgnoreCase("battleship")) {
-            return battleship;
-        } if(shipName.equalsIgnoreCase("submarine")) {
-            return submarine;
-        } if(shipName.equalsIgnoreCase("clipper")) {
-            return clipper;
-        }if(shipName.equalsIgnoreCase("dinghy")) {
-            return dinghy;
-        } else {
-            return null;
+        switch(shipName) {
+            case "aircraftCarrier":
+                return aircraftCarrier;
+            case "battleship":
+                return battleship;
+            case "submarine":
+                return submarine;
+            case "clipper":
+                return clipper;
+            case "dinghy":
+                return dinghy;
+            default:
+                return null;
+        }
+    }
+public int enemyShipOrientation(boolean isEasy, int orientation){// This function returns the proper enemy ship orientation depending on game difficulty.
```

Comment:
Adding a comment describing what these new methods do is very helpful for when other programmers look at your code, good job adding these.